src/Simulation/Algorithms/Kepler/NU2ANOM.hpp

```
@@ -0,0 +1,70 @@
+		// Elliptical orbit (0 < e < 1)
+		else if (e < 1.0 - R_EPSILON) {
+			double sin_e = (glm::sin(nu) * glm::sqrt(1.0 - e * e)) / (1.0 + e * glm::cos(nu));
+			double cos_e = (e + glm::cos(nu)) / (1.0 + e * glm::cos(nu));
+
+			e0 = glm::atan(sin_e, cos_e); // GLM's atan(y, x) overload is equivalent to std::atan2
+			m = e0 - e * glm::sin(e0);
+
+			// Normalize `e0` and `m` to stay within [0, 2pi) rad for elliptical orbits
+			e0 = std::fmod(e0, TWOPI);
+			m = std::fmod(m, TWOPI);
+			if (m < 0.0)
```

Clang (14.0.6)

[warning] statement should be inside braces

(readability-braces-around-statements)